CMakeLists.txt

```diff
@@ -32,8 +33,16 @@ endif()
 
 include(cmake/dependencies.cmake)
 
+if (TELEMETRY_ENABLE_TESTS)
+	include(cmake/googletest.cmake)
+	include(GoogleTest)
+    enable_testing()
```

Comment:
Different indentations. The same also applies to the cmake/googletest.cmake file.